pkgs/by-name/py/pyamlboot/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "pyamlboot";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "superna9999";
+    repo = "pyamlboot";
+    rev = "refs/tags/${version}";
+    hash = "sha256-vpWq8+0ZoTkfVyx+2BbXdULFwo/Ug4U1gWArXDfnzyk=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+  ];
+
+  dependencies = with python3Packages; [
+    pyusb
+    wheel
+    setuptools
+  ];
+
+  nativeInputChecks = [
+    python3Packages.pytestCheckHook
+  ];
+
+  meta = {
+    description = "Amlogic USB Boot Protocol Library";
+    homepage = "https://github.com/superna9999/pyamlboot";
+    license = with lib.licenses; [
+      asl20
+      gpl2Only
+      mit
+    ];
+    maintainers = with lib.maintainers; [ genga898 ];
+    mainProgram = "boot-g12.py";
```

Comment:
Why is upstream boot.py while we are boot-g12.py?

Reply:
It was a mistake from my end.